### PR TITLE
minor language changes for intersection information

### LINKED ISSUE #65: Minor language updates
The largest intersection is ADB, and CERN -> 
The largest intersection is between ADB and Cern
(add between, and ideally remove oxford comma when just 2, but keep when 3)

The largest 5 intersections are Just the empty inter (139), Just ADB -> 
The largest 5 intersections are the empty intersection (139), just ADB
(incorrect "Just"; "inter" should be "intersection; incorrectly capitazlied "Just", and later on there's another oxford comma situation. 
We should also add semicolons between these here, so that it should read: 

The plot is sorted by size in descending order. There are 26 non-empty intersections, all of which are shown in the plot. The largest 5 intersections are the empty intersection (139); Just ADB (23); ADB and CERN (10); ADB and CICA (8); and ADB and APEC (7).

The "Just the empty inter" re-appears later

### PR DESCRIPTION
**Pull request by elizaan**
### Does this PR close any open issues?
Closes #65 

### Have you added or updated relevant tests?
- [ ] Yes
- [ ] No changes are needed

### Have you added or updated relevant documentation?
- [ ] Yes
- [ ] No changes are needed

### Are there any additional TODOs before this PR is ready to go?

TODOs:
- [ ] ...

### CODE REVIEW

File: src/alttxt/tokenmap.py [hunk 1]
```diff
@@ -396,15 +396,19 @@ def max_n_intersections(self, n: int) -> str:
...
-        
-            if i == n - 2:
-                result += "and "
+            result += f"{formatted_names} ({sort[i].size}); "
+
+        result = result.rstrip("; ")
+        parts = result.split("; ")
+
+        if len(parts) > 2:
+            result = "; ".join(parts[:-1]) + "; and " + parts[-1]
```
Comment (JakeWags) [inline src/alttxt/tokenmap.py:405]: Why use `;` instead of `,`?

**Review by JakeWags — CHANGES_REQUESTED** [review]
One minor change.

**Comment by JakeWags** [comment]
Also make sure to add a version bump to `pyproject.toml`

File: src/alttxt/tokenmap.py (hunk 1 above)
Comment (elizaan) [reply src/alttxt/tokenmap.py:405]: https://github.com/visdesignlab/upset-alt-txt-gen/issues/65

that is what the plan for fix of the issue is

**Comment by elizaan** [comment]
> Also make sure to add a version bump to `pyproject.toml`

trying to add trend steepness here also, will do, and might require another review

File: src/alttxt/tokenmap.py (hunk 1 above)
Comment (JakeWags) [reply src/alttxt/tokenmap.py:405]: Thanks for clarifying. I'd suggest filling that information in on the PR description above. Don't need to do it retroactively for this PR though

**Comment by JakeWags** [comment]
> trying to add trend steepness here also, will do, and might require another review

Please ping when this is ready

**Pull request merged**